### DISCUSSION TITLE
github cannot be reached

### DISCUSSION THREAD

**Discussion by Jonathan137** [LMS-Community/slimserver#809] [General]
Get this message (see title) in my log for the docker 8.2.1 version
It looks like the url https://github.com/LMS-Community/lms-plugin-repository/raw/master/extensions.xml does not find a page in the browser. Though https://github.com/LMS-Community/lms-plugin-repository/blob/master/extensions.xml does.

Don't know if this is causing my auto-update for plugins not working?

**Comment by michaelherger** [comment]
What Docker image are you using? What IO::Socket::SSL version is reported in Settings/Information?

And yes, failure to get plugin update information obviously would lead to plugins not updating.

**Comment by Jonathan137** [comment]
Sorry found it, my pihole was blocking https://raw.githubusercontent.com


------------------------------------------------------------------------
Hi Michael,
Thanks for looking into the issue.
See my information below.

IO::Socket::SSL: 2.069
Docker: lmscommunity/logitechmediaserver:stable | 217.1 MB | 2022-08-19 16:49:25

**Comment by mherger** [comment]
Interesting. Is this a rule you set up yourself? Or a recent update to Pi Hole? I'm using it myself and updated only a few days ago.

Thanks for the hint anyway!

**Comment by Jonathan137** [comment]
Dumb enough I have put it myself in the past in the blacklist :-(
Probably due to it was requested from my network a lot.
So it wasn't in an adlist.
